test/e2e/util_gatewayapi_test.go

```diff
@@ -103,19 +103,51 @@ func assertCrdExists(t *testing.T, crdname string) (string, error) {
 	return crdVersion, err
 }
 
+// deleteExistingCRDs deletes if the CRD of the given name exists and returns an error if not.
+func deleteExistingCRDs(t *testing.T, crdname string) error {
```

Comment:
Function is deleting a single CRD, name should reflect:

```suggestion
func deleteExistingCRD(t *testing.T, crdname string) error {
```